vnext/ReactUWP/Views/FlyoutViewManager.cpp

```diff
@@ -300,6 +300,8 @@ void FlyoutShadowNode::AdjustDefaultFlyoutStyle() {
       winrt::FrameworkElement::MaxHeightProperty(), winrt::box_value(50000)));
   flyoutStyle.Setters().Append(
       winrt::Setter(winrt::Control::PaddingProperty(), winrt::box_value(0)));
+  flyoutStyle.Setters().Append(winrt::Setter(
+      winrt::Control::BorderThicknessProperty(), winrt::box_value(0)));
```

Comment:
Once you merge with master you'll probably have to set the border thickness in SetLayoutProps(..) as well.